File: DemoAPI/Models/ClientInfo.cs
public class ClientInfo
{
    public int Id { get; set; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string? Email { get; set; }
    public DateTime Created { get; set; } = DateTime.Now;
    public DateTime LastUpdated { get; set; } = DateTime.Now;
}

File: DemoUI/src/app/models/clientInfo.ts
export class ClientInfo
{
    id!: number;
    firstName: string = "";
    lastName: string = "";
    email: string = "";
    created?: Date;
    lastUpdated?: Date;
    _selected: boolean = false;
    _isBeingEdited: boolean = false;
    _editedBy: string | null = null;

    constructor(init?: Partial<ClientInfo>) {
        Object.assign(this, init);
    }
}